ignite/metrics/accumulation.py

```diff
@@ -73,7 +72,7 @@ def update(self, output: Union[Any, torch.Tensor, numbers.Number]) -> None:
 
         self.accumulator = self._op(self.accumulator, output)
 
-        if hasattr(output, "shape"):
+        if isinstance(output, torch.Tensor):
             self.num_examples += output.shape[0] if len(output.shape) > 1 else 1
         else:
             self.num_examples += 1
```

Comment:
Sure, I changed it to
```python
    def compute(self) -> Tuple[torch.Tensor, int]:
        return (self.accumulator, self.num_examples)
```
because it makes more sense here.